src/main/resources/application.properties

```diff
@@ -1,17 +1,18 @@
 #Server properties
-server.port=9004
+server.port=8080
```

Comment:
Please do not change spring jpa/datasouce default properties

Reply:
They can change manually based on their env right?

Reply:
Yes, but do not touch the defaults. Your `8080` doesn't have to be the default for everyone. Also in the `README.md`  there's a mention of `9004` which you haven't updated.

In general, `8080` conflicts with a variety of processes, so it's best to not use it.

Reply:
OK Then.